<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QWidget" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualization View</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
     <layout class="QHBoxLayout" name="toolbarLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>View Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="viewSelector"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="durationSelector">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>15 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30 min</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomInButton">
        <property name="text">
         <string>üîç+</string>
        </property>
        <property name="toolTip">
         <string>Zoom In</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="zoomOutButton">
        <property name="text">
         <string>üîç-</string>
        </property>
        <property name="toolTip">
         <string>Zoom Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetZoomButton">
        <property name="text">
         <string>‚Ü∫</string>
        </property>
        <property name="toolTip">
         <string>Reset Zoom</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="addSignalButton">
        <property name="text">
         <string>Add Signal...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
